cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -98,6 +99,8 @@ func main() {
 		globalflag.Register(namedFlagSets.FlagSet("generic"), "is-legacy-paravirtual")
 	}
 
+	voptions.AddFlags(namedFlagSets.FlagSet("cloud-node-controller"))
+
```

Comment:
Since node labels are user-provided, suggest to add a validation after parsing, to make sure users are using Kubernetes’ label format and exit early with an error if any key or value is invalid.